mail_tracking/tests/test_mail_tracking.py

```diff
@@ -80,7 +80,9 @@ def test_recipient_address_compute(self):
 
     def test_message_post(self):
         # This message will generate a notification for recipient
-        message = self.env['mail.message'].create({
+        message = self.env['mail.message'].with_context({
+            'mail_notify_force_send': True
+        }).create({
```

Comment:
if we need this here, we need to fix the original problem. The module needs its own tests anyways

Reply:
@hbrunn what is the original problem?

Reply:
@hbrunn tests added.

I don't understand the problem with ```mail_tracking```: ```mail_force_queue``` establishes, as default behaviour, that outgoing email are queued instead of sent immediately (see tests for more clarity). Thus, the test of ```mail_tracking``` must be changed to keep the behaviour unchanged when tested along with ```mail_force_queue```.

The problem described at https://github.com/odoo/odoo/issues/22148 could also be fixed, but the functionality of ```mail_force_queue``` are useful anyway

Reply:
@hbrunn what do you think?

Reply:
the same as before: When `mail_tracking`'s tests fail without this context key, it's a problem. We might be better off just isolating this module then